amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,229 @@
+--Create pet survey in database
```

Comment:
Yup, already noted. Luckly it's a simple fix since there is no db changes, just insertions.